src/views/CVView/CVView.tsx

```diff
@@ -56,13 +58,16 @@ export const CVView = () => {
         <div className="CVView__container">
             <Header/>
             <div className="page__container">
-                <div className="CVView__back">
+                <Grid className="CVView__back"
+                    onClick={() => {
+                        navigate('/list/reserved');
+                    }}>
                     <IoIosArrowDown size={30} className="CVView__back__svg" />
                     <span className="CVView__back__span">Wróć</span>
-                </div>
+                </Grid>
                 <div className="CVView__wrapper">
                     <UserCard
-                        id = {id}
+                        id = {studentId} //@TODO nie podobają mu się typy
```

Comment:
komentarz po angielsku

Reply:
Zamieniłem, choć to jest tylko szybka prośba do grupy o pomysł na rozwiązanie problemu